### PR TITLE
Embed ESMARConf tutorial video

### PR DESCRIPTION
**Pull request by nabury**
Embedded ESMARConf 2023 tutorial video to address issue #26

### CODE REVIEW

**Comment by nabury** [comment]
This is a minor ui update. Could I have a volunteer reviewer @CRSU-Apps/devs

File: ui.R [hunk 1]
```diff
@@ -737,12 +737,19 @@ tabPanel("User Guide",
                             downloadButton("UG", "Download User Guide"),
                             br(),
                             br(),
-                            h4 (tags$strong("Cochrane training Webinar materials", style = "cursive;
+         h4 (tags$strong("ESMARConf 2023 Tutorial", style = "cursive; color: #2196c4")),
```
Comment (RyanJField) [inline ui.R:740]: `style = "cursive;"` is not a valid css styling, for cursive text this should be `style = "font-family: cursive; color: #2196c4"`

File: ui.R [hunk 2]
```diff
@@ -737,12 +737,19 @@ tabPanel("User Guide",
...
+         p(tags$strong(" MetaInsight - an R Shiny web-app for conducting network meta-analysis")),
+         p("A tutorial for MetaInsight v4.0.0 produced for ESMARConf 2023"),
+         HTML('<iframe width="560" height="315" src="https://www.youtube.com/embed/g-RDnQ75Hv4" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>'),
+         br(),
+         br(),
+         h4 (tags$strong("Cochrane training Webinar materials", style = "cursive;
```
Comment (RyanJField) [inline ui.R:746]: Same as above

**Review by RyanJField — CHANGES_REQUESTED** [review]
Looks good see comments.

**Comment by RyanJField** [comment]
Also we should look at making the embedded videos responsive to the size of the browser, but this may be more suited for it's own issue.

This can be achieved by adding a div and responsive bootstrap classes:
```
HTML('<div class = "embed-responsive embed-responsive-16by9"><iframe class = "embed-responsive-item" src="https://www.youtube.com/embed/g-RDnQ75Hv4" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe></div>'),
```

File: ui.R (hunk 1 above)
Comment (nabury) [reply ui.R:740]: I've removed the instances of this from the tab (was copied and pasted from existing code)

**Pull request merged**